### PR TITLE
applications: nrf5340_audio: Update doc regarding bonding

### PR DESCRIPTION
**Pull request by alexsven**
- Document how to clear bonding information

Signed-off-by: Alexander Svensen <alexander.svensen@nordicsemi.no>

### CODE REVIEW

File: applications/nrf5340_audio/README.rst [hunk 1]
```diff
@@ -664,7 +664,10 @@ The application uses the following buttons on the supported development kit:
 |               | * Pressed on the gateway during playback multiple times: Changes the tone frequency.   |
 |               |   The available values are 1000 Hz, 2000 Hz, and 4000 Hz.                              |
 +---------------+----------------------------------------------------------------------------------------+
-| **BTN 5**     | Mutes the playback volume.                                                             |
+| **BTN 5**     | Depending on the moment it is pressed:                                                 |
+|               |                                                                                        |
+|               | * Long-pressed during startup: Clear previously stored bonding information             |
```
Comment (greg-fer) [inline applications/nrf5340_audio/README.rst:669]:
```suggestion
|               | * Long-pressed during startup: Clears the previously stored bonding information        |
```

File: applications/nrf5340_audio/README.rst [hunk 2]
```diff
@@ -1212,6 +1215,8 @@ Complete the following steps to test the CIS mode for one gateway and two headse
    For each button press, the audio stream playback is stopped and the gateway sends a test tone to both headsets.
    These tones can be used as audio cues to check the synchronization of the headsets.
 
+After the kits have paired for the first time, they are now bonded. This means the LTK is stored on each side, and they will only connect to each other unless the bonding information is cleared. Bonding information can be cleared by holding BTN5 during boot.
```
Comment (greg-fer) [inline applications/nrf5340_audio/README.rst:1218]:
```suggestion
After the kits have paired for the first time, they are now bonded.
This means that the Long-Term Key (LTK) is stored on each side, and that the kits will only connect to each other unless the bonding information is cleared.
To clear the bonding information, press and hold **BTN5** during boot.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: applications/nrf5340_audio/README.rst (hunk 1 above)
Comment (erikrobstad) [inline applications/nrf5340_audio/README.rst:669]: Need a period at the end of the sentence

Comment (alexsven) [reply applications/nrf5340_audio/README.rst]: Done

File: applications/nrf5340_audio/README.rst (hunk 2 above)
Comment (alexsven) [reply applications/nrf5340_audio/README.rst]: Fixed

File: applications/nrf5340_audio/README.rst (hunk 1 above)
Comment (alexsven) [reply applications/nrf5340_audio/README.rst]: Fixed

**Review by rick1082 — APPROVED** [review]

File: applications/nrf5340_audio/README.rst [hunk 3]
```diff
@@ -664,7 +664,10 @@ The application uses the following buttons on the supported development kit:
 |               | * Pressed on the gateway during playback multiple times: Changes the tone frequency.   |
 |               |   The available values are 1000 Hz, 2000 Hz, and 4000 Hz.                              |
 +---------------+----------------------------------------------------------------------------------------+
-| **BTN 5**     | Mutes the playback volume.                                                             |
+| **BTN 5**     | Depending on the moment it is pressed:                                                 |
+|               |                                                                                        |
+|               | * Long-pressed during startup: Clears the previously stored bonding information.       |
```
[the remark this answers is not in the document]
Comment (alexsven) [reply applications/nrf5340_audio/README.rst:669]: The information about bonding is written in the 'Testing the default CIS mode' section

**Review by greg-fer — APPROVED** [review]

**Review by koffes — APPROVED** [review]

**Pull request merged**